subprojects/logging/src/test/groovy/org/gradle/internal/deprecation/DeprecationMessagesTest.groovy

```diff
@@ -355,6 +356,6 @@ class DeprecationMessagesTest extends Specification {
     private void expectMessage(String expectedMessage) {
         def events = outputEventListener.events
         events.size() == 1
-        assert events[0].message == expectedMessage
+        assert events[0].message.startsWith(TextUtil.toPlatformLineSeparators("$expectedMessage\n"))
```

Comment:
We need a better adaptation of the tests. This hides problems. For example `DeprecationMessageTest`'s scenarios now print a lot of useless stacktrace.